demos/python_demos/speech_recognition_demo/requirements.txt

```diff
@@ -1,2 +1,4 @@
 numpy
 tqdm
+librosa
```

Comment:
librosa [license](https://github.com/librosa/librosa/blob/main/LICENSE.md)
would be nice to see that in more detailed PR description.
@yssemaev can we use python component with this license?

Reply:
librosa was approved by OSPDT & legal before 2021.1.

Reply:
good then